File: VulpesCloud-launcher/src/main/java/io/github/thecguygithub/launcher/TestCloudClassLoader.java
package io.github.thecguygithub.launcher;

import java.net.URL;
import java.net.URLClassLoader;

public final class TestCloudClassLoader extends URLClassLoader {

    static {
        ClassLoader.registerAsParallelCapable();
    }

    public TestCloudClassLoader() {
        super(new URL[0], ClassLoader.getSystemClassLoader());
    }

    @Override
    public void addURL(URL url) {
        super.addURL(url);
    }
}

File: TestCloud-launcher/src/main/kotlin/io/github/thecguygithub/launcher/TestCloudClassLoader.kt
package io.github.thecguygithub.launcher

import java.net.URL
import java.net.URLClassLoader

class TestCloudClassLoader : URLClassLoader(emptyArray(), ClassLoader.getSystemClassLoader()) {

    companion object {
        init {
            ClassLoader.registerAsParallelCapable()
        }
    }

    public override fun addURL(url: URL) {
        super.addURL(url)
    }


}